src/Sulu/Bundle/WebsiteBundle/Routing/ContentRouteProvider.php

```diff
@@ -116,6 +116,14 @@ public function getRouteCollectionForRequest(Request $request)
                         )
                     );
                 } elseif (
+                    $content->getNodeType() === Structure::NODE_TYPE_EXTERNAL_LINK &&
+                    $content->getNodeState() === StructureInterface::STATE_PUBLISHED
+                ) {
+                    $collection->add(
+                        $content->getKey() . '_' . uniqid(),
+                        $this->getRedirectRoute($request, 'http://' . $content->getPropertyValue('external'))
```

Comment:
I guess the template which is used here is fixed, so this should work. Of course it's not the best solution to hardcode the value here, but that's something we can fix with your refactoring. Just wanted to add this feature without breaking any of the other stuff. Would like to change a lot more here...